### ISSUE TITLE
SCSS compiler error

### ISSUE THREAD

**Issue by tossp** [vmware/clarity#559]
```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

`./src/styles.scss`code
```
@import "~clarity-icons/src/clarity-icons";
@import "~clarity-ui/src/main";
```

`0.8.8` can be compiled successfully

### Actual behavior

```
ERROR in ./~/css-loader?{"sourceMap":false,"importLoaders":1}!./~/postcss-loader!./~/sass-loader!./src/styles.scss
Module build failed:
undefined
^
      File to import not found or unreadable: ../utils/form-variables.
Parent style sheet: D:/src/xx/node_modules/clarity-ui/src/forms/_forms.clarity.scss
      in D:\src\xx\node_modules\clarity-ui\src\forms\_forms.clarity.scss (line 4, column 1)
 @ ./src/styles.scss 4:14-195
 @ multi ./src/styles.scss
```

### Environment details

* **Angular version:** 2.4.9

* **Clarity version:** 0.8.9

* **OS and version:** windows10

* **Browser:** all

**Comment by adityarb88** [comment]
Hi @tossp: We have already added a fix for this which will be published next week in 0.8.10: https://github.com/vmware/clarity/pull/558
This bug was introduced in 0.8.9 so the earlier version (0.8.8) should work fine too.

**Comment by tossp** [comment]
@adityarb88 Thank you very much for your important information
